lib/themes/dosomething/paraneue_dosomething/templates/reportback/reportback-simple-permalink.tpl.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * @file
+ * Reportback confirmation/permalink page for Rogue.
+ */
+?>
+<article class="reportback__permalink">
+  <header role="banner" class="header ">
+    <div class="wrapper">
+      <h1 class="header__title">Awesome! We got your photo.</h1>
+      <p class="header__subtitle">You've entered into the scholarship and your photo will show up in the gallery soon!</p>
```

Comment:
A campaign might not have a scholarship. I think in the other permalink template we only conditionally  show the scholarship language.